src/functions-and-arrays.js

```diff
@@ -78,11 +247,40 @@ const wordsCount = [
   'matter'
 ];
 
-function howManyTimes() {}
+function howManyTimes(array, word) {
+  
+  if (array.length === 0){
+    return 0
+  }
+  let result;
+  let count = 0
+
+  array.forEach(element => {
+    if(element === word){
+      result = count++
+    }
+    if (element !== word){
```

Comment:
Here, you only took into consideration the specific values that the Jasmine tests. And in this case, you should simplify your answers. Meaning that, if you just returned the count, the answer would always be correct and there's no need to specify that three values.
```
function howManyTimes(arrayOfWords, word) {
  let count = 0;
  if (arrayOfWords.length === 0) {
    return 0;
  } else {
    for (i = 0; i < arrayOfWords.length; i++) {
      if (word === arrayOfWords[i]) {
        count++;
      }
    }
    return count;
  }
}
```